astropy/io/fits/hdu/image.py

```diff
@@ -13,7 +13,7 @@
 from ....extern.six import string_types
 from ....extern.six.moves import range, zip
 from ....utils import isiterable, lazyproperty, classproperty, deprecated
-
+from ....wcs import wcs
```

Comment:
You can just do `from ....wcs import WCS` class directly, and then use it straight below.